<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit_calc</class>
 <widget class="QDialog" name="deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>456</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>630</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>630</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit_calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 27, 34);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>10000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>10000</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>6.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(96, 99, 103);
  	color: white; 
	border-radius: 7px;
}
QPushButton:pressed {
    	border: 1px solid  gray;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #A9A9A9, stop: 1 #808080);
}</string>
   </property>
   <property name="text">
    <string>График выплат</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <item>
    <property name="text">
     <string>1 год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <property name="minimum">
    <number>100</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="add_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(96, 99, 103);
  	color: white; 
	border-radius: 7px;
}
QPushButton:pressed {
    	border: 1px solid  gray;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #A9A9A9, stop: 1 #808080);
}</string>
   </property>
   <property name="text">
    <string>+ Добавить пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_money_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(96, 99, 103);
  	color: white; 
	border-radius: 7px;
}
QPushButton:pressed {
    	border: 1px solid  gray;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #A9A9A9, stop: 1 #808080);
}</string>
   </property>
   <property name="text">
    <string>+ Добавить снятие</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sub">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <property name="minimum">
    <number>100</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="sub_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(96, 99, 103);
  	color: white; 
	border-radius: 7px;
}
QPushButton:pressed {
    	border: 1px solid  gray;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #A9A9A9, stop: 1 #808080);
}</string>
   </property>
   <property name="text">
    <string>- Убрать пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sub_money_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(96, 99, 103);
  	color: white; 
	border-radius: 7px;
}
QPushButton:pressed {
    	border: 1px solid  gray;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #A9A9A9, stop: 1 #808080);
}</string>
   </property>
   <property name="text">
    <string>- Убрать снятие</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	border-radius: 7px;
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	border-radius: 7px;
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;

</string>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="displayFormat">
    <string>MM.yyyy</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_add">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="displayFormat">
    <string>MM.yyyy</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_sub">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(61, 64, 67);
color: white; 
border-radius: 7px;
</string>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="displayFormat">
    <string>MM.yyyy</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
